velox/functions/prestosql/aggregates/tests/AverageAggregationTest.cpp

```diff
@@ -594,6 +594,23 @@ TEST_F(AverageAggregationTest, avgDecimalWithMultipleRowVectors) {
   AggregationTestBase::enableTestIncremental();
 }
 
+TEST_F(AverageAggregationTest, avgIntervalWithMultipleRowVectors) {
+  AggregationTestBase::disableTestIncremental();
+
+  auto inputRows = {
+      makeRowVector({makeFlatVector<int64_t>({100, 200}, INTERVAL_DAY_TIME())}),
+      makeRowVector({makeFlatVector<int64_t>({300, 400}, INTERVAL_DAY_TIME())}),
+      makeRowVector({makeFlatVector<int64_t>({500, 600}, INTERVAL_DAY_TIME())}),
```

Comment:
Could you also add some small numbers so that the avg() function actually do the rounding and ensure the rounded result is the same as Presto?

Reply:
1 is the smallest number. I updated 600 to 845. In my updated test case `(100+200+300+400+500+845) / 6 = 390.833`, which is then rounded to 391, instead of truncated. Does the updated test case work for you?

Reply:
Yeah, that looks good to me. You can run a query with the same input data in Presto Java to ensure Java's result is the same.

Reply:
Any idea how to do this? I wasn't able to create an Interval Day smaller than a Day (fraction value). But internally the representation is in miliseconds. I would need a smaller e.g like this one to observe behavior

Reply:
Got it working with seconds! Perhaps the value needs to be exact milisecond not a fraction
```
presto:di> SELECT 
        ->     AVG(c0) AS average_interval
        -> FROM (
        ->     VALUES 
        ->         (INTERVAL '0.1' SECOND),
        ->         (INTERVAL '0.2' SECOND),
        ->         (INTERVAL '0.3' SECOND),
        ->         (INTERVAL '0.4' SECOND),
        ->         (INTERVAL '0.5' SECOND),
        ->         (INTERVAL '0.845' SECOND)
        -> ) AS t(c0);
 average_interval     
------------------
 0 00:00:00.391   
(1 row)
```
I observe same 391 behavior on Java workers